packages/react-hooks/src/useSubscription/useSubscription.ts

```diff
@@ -0,0 +1,55 @@
+import { MedplumClient, SubscriptionEmitter, SubscriptionEventMap } from '@medplum/core';
+import { Bundle } from '@medplum/fhirtypes';
+import { useCallback, useEffect, useRef, useState } from 'react';
+import { useMedplum } from '../MedplumProvider/MedplumProvider.context';
+
+const SUBSCRIPTION_DEBOUNCE_MS = 3000;
+
+export function useSubscription(criteria: string, callback: (bundle: Bundle) => void): void {
+  const medplum = useMedplum() as MedplumClient;
+  const [emitter, setEmitter] = useState<SubscriptionEmitter>();
```

Comment:
Why do we need the emitter?  Why not just call callback directly?

Reply:
Talked about this offline 👍 think we're all good here